src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -29,11 +29,13 @@ const AssigneeSelectorComponent = createReactClass({
   propTypes: {
     id: PropTypes.string.isRequired,
     size: PropTypes.number,
+    memberList: PropTypes.array,
```

Comment:
Can you add a comment here clarifying this behaviour since it might be a little unexpected?